realm/realm-library/src/main/java/io/realm/DynamicRealm.java

```diff
@@ -131,6 +134,7 @@ public void clear(String className) {
      *
      * @param transaction {@link io.realm.DynamicRealm.Transaction} to execute.
      * @throws RealmException if any error happened during the transaction.
+     * @throws Error if checked exception is thrown during transaction execution or committal.
```

Comment:
I merged #1974. Please merge master into `donnfelker:df/doc-exc-javadoc` branch and update the Javadoc comment.